src/2048-expert/index.html

```diff
@@ -13,6 +13,31 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
```

Comment:
nit pick: capitalise to ensure consistency with Liedle and Capidle help dialogues

`HOW TO PLAY 2048 Expert` --> `HOW TO PLAY 2048 EXPERT`